src/daemon/custom_image_host.cpp

```diff
@@ -29,6 +30,7 @@
 #include <QMap>
 #include <QUrl>
 
+#include <future>
```

Comment:
After the refactoring of the parallel request code, is this include still needed?

Reply:
Good catch, I always thought clang-tidy automatically detects unused include, but it turned out that it does not. I will search for other tools on linux which can do this.